hack/verify-govet.sh

```diff
@@ -24,6 +24,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 cd "${KUBE_ROOT}"
 
+go install ./cmd/...
```

Comment:
actually, one nit (somehow missed this) - might be worth adding a comment why this is necessary?